src/nodeTypesStore.js

```diff
@@ -37,6 +41,8 @@ export default new Vuex.Store({
     },
     setPinnedNodes(state, payload) {
       state.pinnedNodeTypes.push(payload);
+      // Remove duplicates
+      state.pinnedNodeTypes = [...new Map(state.pinnedNodeTypes.map(node => [node['type'], node])).values()];
```

Comment:
instead of using a Map since Map are not reactive in this version of vue, use `uniqBy` from `lodash` to determine which objects are duplicated